client/src/components/byline-slot-wrapper/index.jsx

```diff
@@ -0,0 +1,96 @@
+/* global bylineData */
+
+// External dependencies.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+// Internal dependencies.
+import BylineAutocomplete from '../byline-autocomplete';
+import BylineFreeform from '../byline-freeform';
+import BylineList from '../byline-list';
+
+const BylineSlotWrapper = ({
+  profiles,
+  addProfile,
+  removeProfile,
+  reorderProfile,
+  addAuthorLabel,
+  addAuthorPlaceholder,
+  removeAuthorLabel,
+  addFreeformLabel,
+  addFreeformPlaceholder,
+  addFreeformButtonLabel,
+  profilesApiUrl,
+}) => (
+  <div className="components-base-control">
+    {null === profiles ? (
+      <div style={{ textAlign: 'center' }}>
+        <Spinner />
+      </div>
+    ) : (
+      <Fragment>
```

Comment:
Should be able to use `<>` instead of `<Fragment>`

Reply:
It's currently not possible due to the current eslint config.